docs/design-docs/overview.md

```diff
@@ -0,0 +1,144 @@
+---
+id: overview
+---
+
+# Superbench Design
+
+## Goals
+
+Extend current tool to fit distribution test needs in cluster with 100 ~ 1000 nodes, make it modulable, easy to use and easy to scale up.
+Provide more simple and convenient way to:
+1.	Install and deploy the tool in a raw cluster, including bare metal, on-premises, and cloud environments.
```

Comment:
Install and deploy SuperBench in a raw cluster, including bare-metal, on-premises, and cloud environments.